CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/RequestHandler/ExpressionRequestHandler.java

```diff
@@ -0,0 +1,87 @@
+package com.couchbase.CouchbaseLiteServ.server.RequestHandler;
```

Comment:
Do we need this class, as we don't have any test for them?

Reply:
we do not need to cover API tests for expressions?

Reply:
Its ok to leave it in there as its already written. Might need it in the future